internal/view/log_indicator.go

```diff
@@ -107,5 +108,5 @@ func (l *LogIndicator) onOff(b bool) string {
 }
 
 func (l *LogIndicator) update(status string) {
```

Comment:
@ckoehn Thank you for this PR! Good eye!! I think we could refactor the update method as follows:
Does this make sense?

```go
func (l *LogIndicator) update(title string, state bool, pad ...bool) {
	var padding string
	if len(pad) == 0 {
		padding = spacer
	}
	fmt.Fprintf(l, "[::b]%s:%s%s", title, l.onOff(state), padding)
}
```

Reply:
I didn't understand why `pad` is a variadic argument and changed it to a no-variadic one.

Reply:
Yes that's fine. Thank you Christian!